SU2_CFD/src/drivers/CDiscAdjMultizoneDriver.cpp

```diff
@@ -235,9 +235,9 @@ void CDiscAdjMultizoneDriver::Run() {
 
   /*--- Evaluate the objective function gradient w.r.t. the solutions of all zones. ---*/
 
-  SetRecording(NONE, Kind_Tape::OBJECTIVE_FUNCTION_TAPE, ZONE_0);
-  SetRecording(SOLUTION_VARIABLES, Kind_Tape::OBJECTIVE_FUNCTION_TAPE, ZONE_0);
-  RecordingState = NONE;
+  SetRecording(RECORDING::NO_RECORDING, Kind_Tape::OBJECTIVE_FUNCTION_TAPE, ZONE_0);
```

Comment:
see here

Reply:
We can try to make the "NONE" case either expressive or concise, for example the NONE of time marching is steady.
The NONE recording serves to clear indices.

Reply:
We can try to make the "NONE" case either expressive or concise, for example the NONE of time marching is steady.
The NONE recording serves to clear indices.

Reply:
Exactly. For TIME_MARCHING that is done 👍 
Only add a NONE-type option if it not already expressed by another option by an elimination process

Reply:
oh and 👍 for the addition :)

Reply:
What I'm say is RECORDING::NO_RECORDING is not a very good name IMO

Reply:
I'll change it to RECORDING::CLEAR_INDICES (tomorrow). That should be better?
(Sorry I read the comment with a vanished attention span 😴 :) )

Reply:
I think so, gives more info at least, but hey... I'm sleeping on the job too.